README.md

```diff
@@ -146,23 +146,39 @@ see [cockroach-prod](https://github.com/cockroachdb/cockroach-prod)
 
 ## Get in touch
 
-+ cockroach-db@googlegroups.com
-+ \#cockroachdb on freenode
+We spend almost all of our time here on GitHub, and use the [issue
+tracker](https://github.com/cockroachdb/cockroach/issues) for
+bug reports and development-related questions.
 
-## Contributing
+For anything else, message our mailing list cockroach-db@googlegroups.com.
```

Comment:
Make this a link to the archives: https://groups.google.com/forum/#!forum/cockroach-db. Also recommend that people join the mailing list before posting because posts from non-members are held for moderation.